src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json

```diff
@@ -0,0 +1,45 @@
+{
+  "ocr_data_fields":[
+    {
+      "name":"applicant1_firstName",
+      "value":"applicant1_firstName"
+    },
+    {
+      "name":"applicant1_lastName",
+      "value":"applicant1_lastName"
+    },
+    {
+      "name":"applicant1_telephoneNumber",
+      "value":"09999999990"
+    },
+    {
+      "name":"applicant1_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant1_relationToChild",
+      "value": "Step Father"
+    },
+    {
+      "name":"applicant2_firstName",
+      "value":"applicant2_firstName"
+    },
+    {
+      "name":"applicant2_lastName",
+      "value":"applicant2_lastName"
+    },
+    {
+      "name":"applicant2_telephoneNumber",
+      "value":"09999999990"
+    },
+    {
+      "name":"applicant2_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant2_relationToChild",
+      "value": "Step Mother"
```

Comment:
{
      "name": "applicant2_relationToChild",
      "value": "Step Father"
    }, pls this , which not required for ur tcase type

Reply:
This added in order to test optional fields for A60